malaria-ios/NSManagedObjects/RestAPI/CollectionRFQuestions.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+import CoreData
+
+/**
+ A collection of questions for the
+ Rapid Fire game.
+ */
+
+public class CollectionRFQuestions: NSManagedObject {
+  
+  @NSManaged public var posts: NSSet
```

Comment:
In my opinion, the questions shouldn't be retrieved as a "post"

Therefore they wouldn't have the same fields as the other endpoints